db/models/stigData.ts

```diff
@@ -1,19 +1,21 @@
 /* eslint-disable no-use-before-define */
 import {
   Model,
-  InferAttributes,
-  InferCreationAttributes,
   DataTypes,
-  CreationOptional,
-  NonAttribute,
   Association,
+  type InferAttributes,
+  type InferCreationAttributes,
+  type CreationOptional,
+  type NonAttribute,
+  type BelongsToManyAddAssociationMixin,
+  type BelongsToManyRemoveAssociationMixin,
 } from "sequelize";
 
 import { AssessmentItem } from "./assessmentItem";
 import { Stig } from "./stig";
 import { StigIdent } from "./stigIdent";
 import { EvaluationItem } from "./evaluationItem";
-import type { Override } from ".";
+import type { Override, StigReference, StigResponsibility } from ".";
```

Comment:
Why are we pulling from index.ts instead of their files where they're actually defined?  And also why are we importing them as types?